invenio_orcid/config.py

```diff
@@ -26,6 +26,8 @@
 
 ORCID_SYNCHRONIZATION_ENABLED = True
 
+ORCID_JSON_CONVERTER_MODULE = 'invenio_orcid.utils:convert_to_orcid'
```

Comment:
do you mean this https://github.com/inspirehep/inspire-next/blob/master/inspirehep/modules/records/serializers/orcid_serializer.py ?